uniffi_bindgen/src/bindings/kotlin/templates/StringHelper.kt

```diff
@@ -38,6 +46,7 @@ public object FfiConverterString: FfiConverter<String, RustBuffer.ByValue> {
     }
 
     override fun write(value: String, buf: ByteBuffer) {
+        check(value)
```

Comment:
ouch - is there really no way to implement this other than to do one full pass over the string to check for the invalid sequence, then another to actually perform the encoding?

Reply:
Done, with the help of @linabutler.